package/com.unity.formats.usd/Runtime/Plugins/x86_64/Linux.meta

```diff
@@ -1,8 +0,0 @@
-fileFormatVersion: 2
```

Comment:
Linux and OSX support seem to be removed, is that intentional?

Reply:
Overlook on my part until I noticed the failed CI jobs (thanks @clusty) 
I'm setting up bokken jobs to build usd and the bindings on all 3 platforms. I'll update the PR asap.